roles/search_enterprise/molecule/default/verify.yml

```diff
@@ -25,7 +25,6 @@
           Refreshing accepted content media-type cache, accepted type count: ([7-9]\d|[1-9]\d{2,})$
       ansible.builtin.assert:
         that:
-          - not error_log.stdout | regex_search(' WARN ')
```

Comment:
I suppose a new warning has appeared which we cannot do much with?

Reply:
indeed see the related commit message